### ISSUE TITLE
Percent signs in PptCharts cause unrepairable failure

### ISSUE THREAD

**Issue by bkanber** [PHPOffice/PHPPresentation#452]
When using PowerPoint2007 writer to create charts and opening in Powerpoint for Mac, I get an unrepairable error from PPT. 

The source of the issue is two lines in PptCharts: 

Line 108: 

```
$objWriter->writeElementIf($hPercent != null, 'c:hPercent', 'val', $hPercent . '%');
```

Line 2325:

```
$objWriter->writeAttribute('val', '100%');
```

If I remove the two percent symbols everything works as expected.

Would this be a safe pull request to make?

**Comment by MindlineAnalytics** [comment]
Yes its because of the new version of the powerpoint. This is safe.

**Comment by mhasanshahid** [comment]
Yes its because of the new version of the powerpoint. This is safe.

**Comment by bkanber** [comment]
Thank you, I have submitted a pull request with passing tests. Please let me know if the changes I made to the test suite are acceptable. I have left the following comment on the PR explaining the logic: 

> Regarding the changes I made to the test schema: I looked up the schema here http://www.datypic.com/sc/ooxml/s-dml-chart.xsd.html and added the xsd:unsignedShort to the union statement, as both of the types in question are listed as being derived from xsd:unsignedShort. I have left in the percent sign types in the union, figuring that this would allow for some backwards compatibility in validation, however let me know if you would like me to update that schema. I'm just not really sure what it's properly supposed to be.

**Comment by Progi1984** [comment]
Fixed in  #453
